java/messages/src/main/java/com/nedap/retail/messages/epc/v2/approved_difference_list/ApprovedDifferenceListWithBarcodes.java

```diff
@@ -0,0 +1,7 @@
+package com.nedap.retail.messages.epc.v2.approved_difference_list;
+
+import java.util.List;
+
+public class ApprovedDifferenceListWithBarcodes extends ApprovedDifferenceList {
+    public List<String> barcodes;
```

Comment:
isn't it more logical to include the barcode in a new `GtinDifferenceWithBarcode`?